pandas/_libs/tslibs/offsets.pyx

```diff
@@ -559,6 +607,20 @@ cdef class BaseOffset:
 
     @cache_readonly
     def freqstr(self) -> str:
+        """
+        Return a string representing the frequency.
```

Comment:
How does this differ from the `name`? i.e. is there a diff between,

```
>>> pd.offsets.Hour(5).name  # which is "H" in above example
>>> pd.offsets.Hour(5).freqstr
```
?

Reply:
AFAIU: `name` is the "base" frequency unit (H), and `freqstr` is the entire frequency with its multiple (5H).

Added additional examples to `name` and `freqstr` to clarify this